core/gui/src/app/common/service/user/auth.service.ts

```diff
@@ -131,8 +134,15 @@ export class AuthService {
     const role = this.jwtHelperService.decodeToken(token).role;
     const email = this.jwtHelperService.decodeToken(token).email;
     if (this.config.env.inviteOnly && role == Role.INACTIVE) {
-      alert("The account request of " + email + " is received and pending.");
-      this.gmailService.notifyUnauthorizedLogin(email);
+
+      this.modal.confirm({
+        nzTitle: "You Need Access",
+        nzContent: "Access to the platform is restricted. Please request access from an admin or switch to an account that already has access.",
```

Comment:
"Access to the platform is restricted" ==> "Currently the platform is invitation-only"
"Please request access from an admin" ==> "Please request access from the platform admin"